vibe_code_apps_20/tests/test_full_mcp_default.py

```
@@ -0,0 +1,85 @@
+        "via": "mcp-exec",
+        "src": str(src),
+        "dst": str(dst),
+        "lights_w_per_m2": 4.5,
+        "equip_w_per_m2": 4.2,
+        "infil_mult": 1.4,
+        "hint": "x",
+    }
+
+    real_import = builtins.__import__
+
+    def _imp(name, globals=None, locals=None, fromlist=(), level=0):
```

Ruff (0.15.21)

[error] Function argument `globals` is shadowing a Python builtin

(A002)